waspstation/code/game/gamemodes/endgame/endgame.dm

```diff
@@ -0,0 +1,82 @@
+
+#define CASCADE_EVENT_COUNTDOWN 54000 // 1 hour 30 minutes
+#define CASCADE_SATURATION_COUNTDOWN 9000 // 15 minutes
+
+#define CINEMATIC_BLUESPACE_CATACLYSM 15
+/datum/cinematic/bluespace_cataclysm
+	id = CINEMATIC_BLUESPACE_CATACLYSM
+	is_global = TRUE
+	cleanup_time = 300
+
+/datum/cinematic/bluespace_cataclysm/New()
+	screen = new(src)
+
+/datum/cinematic/bluespace_cataclysm/content()
+	flick("intro_cataclysm", screen)
+	sleep(30)
```

Comment:
can this be relegated to a timer?

Reply:
That's the precedent in other cinematics